app/views/providers/ukprns/edit.html.erb

```diff
@@ -0,0 +1,30 @@
+<% page_title = t("helpers.label.provider.ukprn") %>
+
+<%= content_for :page_title, title_with_error_prefix(page_title, @form_object.errors.present?) %>
+
+<% content_for :before_content do %>
+  <%= govuk_back_link_to(details_provider_recruitment_cycle_path(@provider.provider_code, @provider.recruitment_cycle_year)) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with(
+      model: @form_object,
+      url: provider_recruitment_cycle_ukprn_path(@provider.provider_code, @provider.recruitment_cycle_year),
+      method: :post,
+      scope: "",
+      builder: GOVUKDesignSystemFormBuilder::FormBuilder,
+    ) do |f| %>
+
+      <%= f.govuk_error_summary %>
+
+      <h1 class="govuk-heading-l">
+        <span class="govuk-caption-l"><%= @provider.provider_name %></span>
+      </h1>
+
+      <%= f.govuk_text_field(:ukprn, label: { text: page_title, size: "m" }, width: 10, data: { qa: "ukprn" }) %>
```

Comment:
```suggestion
      <%= f.govuk_text_field(:ukprn,
        caption: { text: @provider.provider_name, size: "l" },
        label: { text: page_title, size: "l", tag: "h1" },
        width: 10,
        data: { qa: "ukprn" }) %>
```

Reply:
We have since add another field `urn` which makes the argument for the heading to sit above the two fields. Example: 

<img width="905" alt="Screenshot 2021-07-02 at 16 31 39" src="https://user-images.githubusercontent.com/28728/124297494-0a821d80-db53-11eb-8122-1e5b319754c1.png">

Let me know what you think.